### DISCUSSION TITLE
I don't Have School issued Email, But I have documents but i am unable to continue.

### DISCUSSION THREAD

**Discussion by Mayuresh1611** [community/community#50410] [GitHub Education]
Previously on 12th march I successfully enrolled in GitHub global campus. I did not have a school-issued email but I uploaded the necessary documents. And got successfully approved. But this morning I checked my emails and it said to reverify your account. So I went to reverify the account but as I don't have a school-issued email so I tried using the normal one ( as I previously did ). I am unable to continue further document upload process ( because I have the documents ) (Colleges here don't give school-issued emails ), I want to be part of GitHub global student campus. Looking forward to a solution...

**Comment by mdjvd** [comment]
I have the same issue. What are we supposed to do if we are not provided with a school-issued email?
